packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html

```diff
@@ -1,247 +1,179 @@
+                                'one': ('ip_alerts_spam_1' | translate),
+                                'other': ('ip_alerts_spam_other' | translate: { count: {} })
+                            }"
                     >
-                        <span
-                            data-ng-if="!singleService()"
-                            data-translate="ip_alerts_none"
-                        >
-                        </span>
-                        <span
-                            data-ng-if="singleService()"
-                            data-translate="ip_alerts_service_none"
-                        >
-                        </span>
-                    </div>
-                </div>
+                    </span>
+                    <button
+                        class="btn btn-link"
+                        type="button"
+                        data-ng-if="!singleService()"
+                        data-ng-repeat="ip in alerts.spam"
+                        data-ng-click="navigateToService(ip.ip.routedTo.serviceName)"
+                        data-ng-bind="ip.ip.routedTo.serviceName"
+                    ></button>
+                </p>
 
-                <oui-spinner data-ng-if="loading.alerts" size="s">
-                </oui-spinner>
+                <p data-ng-if="alerts.antihack.length">
+                    <span
+                        class="d-block"
+                        data-ng-pluralize
+                        data-count="alerts.antihack.length"
+                        data-when="{
+                                '0': ('ip_alerts_antihack' | translate),
+                                'one': ('ip_alerts_antihack_1' | translate),
+                                'other': ('ip_alerts_antihack_other' | translate: { count: {} })
+                            }"
+                    >
+                    </span>
+                    <button
+                        class="btn btn-link"
+                        type="button"
+                        data-ng-if="!singleService()"
+                        data-ng-repeat="ip in alerts.antihack"
+                        data-ng-click="navigateToService(ip.ip.routedTo.serviceName)"
+                        data-ng-bind="ip.ip.routedTo.serviceName"
+                    ></button>
+                </p>
 
-                <h4 data-translate="ip_addresses_grid_header"></h4>
+                <p data-ng-if="alerts.arp.length">
+                    <span
+                        class="d-block"
+                        data-ng-pluralize
+                        data-count="alerts.arp.length"
+                        data-when="{
+                                '0': ('ip_alerts_arp' | translate),
+                                'one': ('ip_alerts_arp_1' | translate),
+                                'other': ('ip_alerts_arp_other' | translate: { count: {} })
+                            }"
+                    >
+                    </span>
```

Comment:
**nitpick:** Can we replace `<span>` + `.d-block` with the block level elements?